modules/hub/hub.go

```diff
@@ -152,6 +152,40 @@ func IsPullRequest(issue *github.Issue) bool {
 	return true
 }
 
+func CreateWebhook(token string, repo *repository.RepositoryStruct, secret string, url string) error {
```

Comment:
__[golint]__
* [golint] exported function CreateWebhook should have comment or be unexported